sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -170,15 +187,39 @@ export interface RecognizeLinkedEntitiesSuccessResult extends TextAnalyticsSucce
     readonly entities: LinkedEntity[];
 }
 
+// @public (undocumented)
+export interface SentenceAspect {
+    confidenceScores: AspectConfidenceScoreLabel;
+    relations: AspectRelation[];
+    sentiment: SentenceAspectSentiment;
+    text: string;
+}
+
+// @public
+export type SentenceAspectSentiment = "positive" | "mixed" | "negative";
+
+// @public (undocumented)
+export interface SentenceOpinion {
+    confidenceScores: AspectConfidenceScoreLabel;
+    isNegated: boolean;
+    sentiment: SentenceOpinionSentiment;
+    text: string;
+}
+
+// @public
+export type SentenceOpinionSentiment = "positive" | "mixed" | "negative";
+
 // @public
 export interface SentenceSentiment {
+    aspects?: SentenceAspect[];
     confidenceScores: SentimentConfidenceScores;
+    opinions?: SentenceOpinion[];
     sentiment: SentenceSentimentLabel;
-    text?: string;
+    text: string;
```

Comment:
Is making this property required a breaking change?

Reply:
I am not sure why it was optional in the first place. This is supposed to be a response object. I am following up with the service team.

Reply:
@joheredi they confirmed that text being optional was a mistake in the stable version and they are fixing it https://github.com/Azure/azure-rest-api-specs/pull/10179. I added a line to the CHANGELOG.md making it a breaking change.